src/Metadata/Property/Factory/InheritedPropertyMetadataFactory.php

```diff
@@ -0,0 +1,54 @@
+<?php
```

Comment:
Why try to achieve a 100% coverage in unit tests to be able to use each library (metadata, Swagger, Hydra...) as separate components. Btw unit test allows to better think the public API of classes than functional tests.